cmd/cstorvolumeclaim/start.go

```diff
@@ -93,16 +101,37 @@ func Start(controllerMtx *sync.RWMutex) error {
 
 	// blocking call, can't use defer to release the lock
 	controllerMtx.Unlock()
-
 	if err != nil {
 		return errors.Wrapf(err, "error building controller instance")
 	}
 
-	go kubeInformerFactory.Start(stopCh)
-	go cvcInformerFactory.Start(stopCh)
-
 	// Threadiness defines the number of workers to be launched in Run function
-	return controller.Run(2, stopCh)
+	run := func(context.Context) {
+		// run...
+		stopCh := make(chan struct{})
+		kubeInformerFactory.Start(stopCh)
```

Comment:
why these are not in go routines now?

Reply:
This will help to make sure kube and CVC Informers are started before staerting the main controller

Reply:
Even [controller-runtime](https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/leaderelection/leader_election.go)  uses the same implementation using client-go